at.vintagestory.VintageStory.metainfo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>at.vintagestory.VintageStory.unstable</id>
```

Comment:
this should match the app-id. It doesn't

Reply:
I just double checked and it does appear to match the app-id. Is it possible for you to look again? Apologies if I am missing the reference somewhere.

I've compared 
https://github.com/flathub/flathub/blob/d117c67117bfe14ea0cd28b44aef681a13bc9714/at.vintagestory.VintageStory.metainfo.xml#L3

to
https://github.com/flathub/flathub/blob/d117c67117bfe14ea0cd28b44aef681a13bc9714/at.vintagestory.VintageStory.metainfo.xml#L3

Reply:
also why the `unstable`?

Reply:
I'll resolve the filenames now. Thank you.

The name is unstable because it is pointed at the unstable branch for VintageStory. The stable version is already on Flathub (see link in the PR template above).

Would you prefer I follow a different naming convention? Happy to alter however desired.

Reply:
just use the beta branch of the existing package to be on flathub-beta.

in general flathub is not for "unstable" software.